pr-review.js

```
@@ -193,10 +196,11 @@ async function approvePullRequest(body = "LGTM! Approving.") {
   } catch (error) {
     if (error.response && error.response.status === 401) {
       console.error("Error: Unauthorized. The GITHUB_TOKEN may be invalid or expired.");
-    } else {
-      console.error(`Error approving PR: ${error.message}`);
     }
-    console.error(`Error approving PR: ${error.name} ${error.message}`);
+    let errorMsg = `Error approving PR: ${error.message}`;
+    if (error.errors) errorMsg += ` | errors: ${JSON.stringify(error.errors)}`;
+    if (error.status) errorMsg += ` | status: ${error.status}`;
+    console.error(JSON.stringify('Approve error: ', error));
```

Issue: The use of `JSON.stringify` with a string and an error object is incorrect and will not produce the intended output. It will stringify the string and ignore the error object.

**Suggestion:** Use template literals or string concatenation to properly format the error message: `console.error('Approve error:', error);`